library/tasks/TagGeneExon.wdl

```diff
@@ -1,11 +1,27 @@
 task TagGeneExon {
-  # this task requires a gtf file with no internal comments
-  # each record must have a gene_name and transcript_name in addition to a gene_id and transcript_id
-  # the annotation file must be in gtf format (terminal semicolon) not gff format.
-  # the gtf must not have any terminal white space at the end of any line
   File annotations_gtf
   File bam_input
-  Int estimated_required_disk = ceil((size(bam_input, "G") + size(annotations_gtf, "G")) * 2)
+
+  # runtime values
+  String docker = "quay.io/humancellatlas/secondary-analysis-dropseqtools:v0.2.2-1.12"
+  Int machine_mem_mb = 7500
+  Int cpu = 1
+  Int disk = ceil((size(bam_input, "G") + size(annotations_gtf, "G")) * 2)
+  Int preemptible = 0
+
+  meta {
+    description: "AMBROSE HALP!!"
+  }
+
+  parameter_meta {
+    annotations_gtf: "each record must have a gene_name and transcript_name in addition to a gene_id and transcript_id.  Must be in gtf format"
```

Comment:
Could also be useful to include the note that "the gtf must not have any terminal white space at the end of any line"

Reply:
oh nice catch, im bad at copying things =/